docs/architecture.md

```
@@ -28,13 +29,14 @@
-  │ 1. $GITHUB_ACTION_PATH から中央 templates の絶対パスを解決
-  │ 2. caller root に config があれば優先，無ければ中央 templates/ を採用
-  │ 3. scripts/generate-textlint-runtime.py で prh の絶対パスを埋め込んだ
+  │ 1. PR に 👀 reaction を付け「review 開始」を可視化（`pull_request` イベント時のみ）
+  │ 2. $GITHUB_ACTION_PATH から中央 templates の絶対パスを解決
+  │ 3. caller root に config があれば優先，無ければ中央 templates/ を採用
+  │ 4. scripts/generate-textlint-runtime.py で prh の絶対パスを埋め込んだ
   │    .textlintrc.runtime.json を生成
-  │ 4. Node.js setup
-  │ 5. reviewdog/action-markdownlint → PR レビューコメント
-  │ 6. textlint を tmpdir に install して実行 → reviewdog で PR レビューコメント
+  │ 5. Node.js setup
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)